src/cli.ts

```diff
@@ -154,6 +156,16 @@ if(typeof argv['animation-time-budget'] === 'string') {
     }
 }
 
+let traceFilename;
+if(typeof argv['trace-filename'] === 'string') {
```

Comment:
lint?

Reply:
I'd really like to set up an automated formatter, but the (inconsistent) style here matches upstream.